### ISSUE TITLE
Wanted: co-maintainer / second maintainer

### ISSUE THREAD

**Issue by jonascj** [sphinx-contrib/images#20]
**TL; DR;** A second maintainer would be welcome, if you're interested let me know. Until summer 2021 I have limited time to dedicated to merging PRs and handling issues.

I only intended to help complete spinus#56 by leaving a comment, to gain Sphinx 2.x support which I needed at the time, but to move forward the project ended up under the https://github.com/sphinx-contrib umbrella with me as admin (on https://pypi.org/project/sphinxcontrib-images/ as well).

There are some fine issues waiting to be fixed and some fine looking PRs to fix some of the issues, but I have had very little time to devote to the project since May 2020 and I will continue to have little time until late spring or early summer 2021.

Adding a second maintainer or co-maintainer could help move the project forward at a more steady pace. The intention behind joining the https://github.com/sphinx-contrib repo was to ensure continued development, since it would be easy for others to take over.

Let me know if you're interested. Tasks would include one or more of answering issue tickets, reviewing PRs, merging PRs, releasing to PyPi. I'm no expert in any of this, but I'll be happy to help get new co-maintainers started on the tasks.

**Update June 2023**
I am still looking for a second maintainer as of June 2023 - regardless of my time to invest in the project it would be smart to have a second person with knowledge of the project.

There are still some fine issues to fix and support for figures to be added. I do not use the extension at the moment, because figures are not supported, but support can be added and I'd like to do so (or have someone else do it of course).

**Comment by SilverRainZ** [comment]
Hello, I wanna to be the co-maintainer :D

**Comment by jack1142** [comment]
It's just a thought, but perhaps people at [Executable Books](https://github.com/executablebooks) would be interested in this? You might want to read about their project and ask one of the org's members if you think it could work for you.

**Comment by jonascj** [comment]
@jack1142 Great idea. I think I'll wait to reach out though, hoping a contributor will invite him-/herself.

**Comment by 12rambau** [comment]
I'm not sure I'm fit for the job but I currently build up my skills in maintaining python libs so I would be happy to help you if it's still on your schedule. I will obviously need a tour of the lib to better understand what I'm dealing with. 
Anyway let me know if you pass by Github ;-)

**Comment by 12rambau** [comment]
Hi @jonascj I saw that some new issues were opened are you still looking for a co-maintainer ?

**Comment by 12rambau** [comment]
Let me do some preparation PR first and I'll let you know when I'm ready to make a test release

**Comment by jonascj** [comment]
@12rambau Sure and let us move discussion of the next release to #33

**Comment by terencehonles** [comment]
@jonascj I can possibly help a little, but maybe this could be a @jazzband project? Not sure if you looked into @executablebooks as was mentioned previously

I've added #41 which should help get us closer to a next release. I can test the wheel against our docs (which we very minimally use the plugin) to ensure it does indeed build correctly, but examining the wheel it _did_ look like it packaged the right things and it does build the test docs.
